DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -570,9 +580,19 @@ static function bool BingoGoalImpossible(string bingo_event, int start_map, int
         break;
 
     case 6: // Hong Kong
-    case 7:// fallthrough to 2nd half of Hong Kong
         switch(bingo_event)
         {
+        // // these two goals can actually be done with the way these starts currently work, but would normally be impossible
+        // case "ClubEntryPaid":
+        // case "M06JCHasDate":
+        //     return start_map > 65;
+        }
+    case 7: // fallthrough to 2nd half of Hong Kong
+        switch(bingo_event)
+        {
+        case "MaggieCanFly":
+        case "PoliceVaultBingo": // TODO: remove once a datacube with the vault code is added
+            return start_map > 70;
```

Comment:
```
        case 65:// start here with Have_Evidence
            friendlyName = "Hong Kong (Lucky Money)";
            return "06_HongKong_WanChai_Underworld";
```
in start 65 you already have the DTS

```suggestion
            return start_map >= 65;
```

Reply:
Yes, but Maggie can still be there all the way up until you complete Versalist 2, even with a 61 ("Wan Chai Market") start. I'm not sure what the criteria are to make her disappear, but I carefully tested not talking to her until after every goal.